### PR TITLE
User profile page dat

### PR DESCRIPTION
**Pull request by atkinson-daniel**
Complete Use Stories 19 - 21
1) Updated the user profile page to show user info (except for the password)
2) Created an edit user info form 
3) Create a PasswordsContoller 
4) Create a view page for user to edit their password

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -9,8 +9,8 @@ class User < ApplicationRecord
                         :zip_code,
                         :email_address
 
-  validates_presence_of :password, require: true
-
+  # add merchants controller(new create index show edit update destroy)
+  # on create length validation rails for password
   enum role: {user: 0, merchant: 1, admin: 2}
```
Comment (StephanieFriend) [inline app/models/user.rb:14]: Awesome job figuring this out!

File: config/routes.rb [hunk 2]
```diff
@@ -2,13 +2,7 @@
...
   root 'welcome#index'
 
-  get "/merchants", to: "merchants#index"
-  get "/merchants/new", to: "merchants#new"
-  get "/merchants/:id", to: "merchants#show"
-  post "/merchants", to: "merchants#create"
-  get "/merchants/:id/edit", to: "merchants#edit"
-  patch "/merchants/:id", to: "merchants#update"
-  delete "/merchants/:id", to: "merchants#destroy"
+  resources :merchants
```
Comment (margoflewelling) [inline config/routes.rb:5]: Nice! Good use case for resources.

Comment (StephanieFriend) [inline config/routes.rb:5]: Nice!

File: spec/features/users/edit_spec.rb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    user.reload
+
+    expect(current_path).to eq('/user/profile')
+    expect(page).to have_content("Logged in as #{user.name}")
+    expect(page).to have_content(user.name)
+    expect(page).to have_content(user.street_address)
+    expect(page).to have_content(user.city)
+    expect(page).to have_content(user.state)
+    expect(page).to have_content(user.zip_code)
+    expect(page).to have_content(user.email_address)
```
Comment (StephanieFriend) [inline spec/features/users/edit_spec.rb:36]: Nice testing!

**Review by StephanieFriend — APPROVED** [review]
Great testing! Awesome job figuring out the password edit!

**Pull request merged**